packages/core/src/utilities/calculateSpacingBetweenImageIds.ts

```diff
@@ -142,7 +156,7 @@ export default function calculateSpacingBetweenImageIds(
       console.debug(
         'Could not calculate spacing. The VolumeViewport visualization is compromised. Setting spacing to 1 to render'
       );
-      spacing = 1;
+      spacing = DEFAULT_SPACING;
     }
```

Comment:
I would use the same logic here for spacing using the spacing between columns, and drop an error message.  This is actually quite a serious issue in that it can be non obvious what is going on.    If you don't want to touch this one right now, then just leave it spacing = 1;